doc/containers.rst

```diff
@@ -0,0 +1,46 @@
+.. _containers:
+
+Containers
+==========
+
+We provide a pyPESTO docker image in OCI format through `docker.io/stephanmg/pypesto`.
+
+Develop versions are pushed as tag ``latest``.
```

Comment:
Are they? I don't see any at https://hub.docker.com/r/stephanmg/pypesto.

Also, the workflow uses `ICB_DCM/pypesto`.

Reply:
Should be also published now, manually pushed latest image with commit from `develop` latest from today: commit sha 20fd9dec68ff796f5be0aa38de182be4b1d00563